features/bicycle/ref.feature

```diff
@@ -25,8 +25,8 @@ Feature: Bike - Way ref
             | ab    |      | E7  |
 
         When I route I should get
-            | from | to | route |
-            | a    | b  | E7,E7 |
+            | from | to | route                               | ref   |
+            | a    | b  | {highway:primary},{highway:primary} | E7,E7 |
```

Comment:
NCN and RCN routes are mostly tagged with route relations now rather than the older `ncn_ref` and `rcn_ref` tags on ways, though `lcn_ref` is still frequently used for city cycle routes IME.